doc/7/getting-started/react/standalone/index.md

```diff
@@ -0,0 +1,121 @@
+---
+type: page
+code: false
+title: Standalone
+description: Getting started with Kuzzle and ReactJS
+order: 0
+---
+
+# Getting Started with Kuzzle and ReactJS
+
+This section deals with **Kuzzle V2** (+ **Javascript SDK 7**) and **ReactJS**. We will create **documents** in Kuzzle and subscribe to
+[document notifications](/sdk/js/7/essentials/realtime-notifications#document-messages) to develop a realtime chat.
+
+## Requirements
+
+- **Node.js** >= 12.0.0 ([install here](https://nodejs.org/en/download/))
+- **Create React App** ([install here](https://github.com/facebook/create-react-app))
+- **Running Kuzzle V2 Stack** ([instructions here](/core/2/guides/getting-started/running-kuzzle))
+
+## Prepare your environment
+
+Create your React app and install all the dependencies from the command line using npm:
+```bash
+npx create-react-app kuzzle-playground
+```
+Install Kuzzle's Javascript SDK: 
+```bash
+cd kuzzle-playground
+npm install kuzzle-sdk@7
+```
+
+We'll rewrite the src/App.js so you can remove everything inside.
+
+## Instantiating Kuzzle SDK
+
+We have to connect the server so that our client can interact with it.
+To do this, we have to create a *src/services/kuzzle.js* file to put our kuzzle instance declaration:
+
+<<< ./snippets/kuzzle.js.snippet[js]
+
+You can now edit the *src/App.js* file to connect to Kuzzle. To do this, import the kuzzle service file and the following imports in the same time:
+
+<<< ./snippets/App.js.snippet:1[js]
+
+Now, add the app class and add in the constructor a `message` property in the state (we'll use it to store the user input message) a `username` property, `messages` property (we’ll use it to store all messages) and a `validate` property (Value that will change the display):
```

Comment:
IMHO instructions are unclear here because the boilerplate is shipped with a function called `App` inside this file